x/distribution/client/cli/query.go

```diff
@@ -143,7 +143,11 @@ func GetCmdQueryValidatorSlashes(queryRoute string, cdc *codec.Codec) *cobra.Com
 				return err
 			}
 
-			var slashes types.ValidatorSlashEvent
+			if len(res) < 3 {
```

Comment:
Is this really what we want to return? Returning an empty list - `[]` - will be easier to machine-parse and is semantically correct.